### PR TITLE
Add a note about hooks in the CRUDController

### PR DESCRIPTION
**Pull request by Nek-**
I am targeting this branch because it's the one everybody read on the Symfony doc.

## Content

Just some doc for hooks in the `CRUDController`.

### CODE REVIEW

**Comment by OskarStark** [comment]
A Code example would be Great

File: Resources/doc/reference/saving_hooks.rst [hunk 1]
```diff
@@ -106,4 +106,32 @@ The service declaration where the ``UserManager`` is injected into the Admin cla
             </call>
         </service>
 
+
+Hooking in the Controller
+-------------------------
+
+You may have notice that the hooks present in the **Admin** doesn't allow you
```
Comment (greg0ire) [inline Resources/doc/reference/saving_hooks.rst]: - "notice" => "noticed"
- "doesn't" => "do not"

File: Resources/doc/reference/saving_hooks.rst [hunk 2]
```diff
@@ -106,4 +106,32 @@ The service declaration where the ``UserManager`` is injected into the Admin cla
...
+to interact with the process of deletion: you can't cancel it. To achieve this
+you should be aware that there is also a way to hook on actions in the Controller.
+
+If you defines a custom controller that inherits from ``CRUDController``, you can
```
Comment (greg0ire) [inline Resources/doc/reference/saving_hooks.rst]: "defines" => "define"

File: Resources/doc/reference/saving_hooks.rst [hunk 3]
```diff
@@ -106,4 +106,32 @@ The service declaration where the ``UserManager`` is injected into the Admin cla
...
+redefine the following methods:
+
+- new object : ``preCreate($object)``
+- edited object : ``preEdit($object)``
+- deleted object : ``preDelete($object)``
+- show object : ``preShow($object)``
+- list objects : ``preList($object)``
+
+If theses methods return a **Response**, the process is interupted and the response
```
Comment (greg0ire) [inline Resources/doc/reference/saving_hooks.rst]: - "theses" => "these"
- "interupted" => "interrupted"

File: Resources/doc/reference/saving_hooks.rst [hunk 4]
```diff
@@ -106,4 +106,32 @@ The service declaration where the ``UserManager`` is injected into the Admin cla
...
+will be return as it by the controller (if it return null, the process continues). You
```
Comment (greg0ire) [inline Resources/doc/reference/saving_hooks.rst]: - "return" => "returned"
- "as it" => "as is"
- "return" => "returns"

**Comment by Nek-** [comment]
@OskarStark Controller overriding deserves its own documentation page IMO. That's why I didn't add an example in this context.

**Comment by OskarStark** [comment]
> @OskarStark Controller overriding deserves its own documentation page IMO. That's why I didn't add an example in this context.

Are you willing to create one?

**Comment by core23** [comment]
Thanks @Nek-

**Pull request merged**